webv2/uniqueid/uniqueid_assignment.go

```diff
@@ -0,0 +1,219 @@
+				}
+
+				for sourcei, sourceindexes := range sourcetable.Indexes {
+
+					for spanneri, spannerindexes := range spannertable.Indexes {
+
+						if sourceindexes.Name == spannerindexes.Name {
+
+							indexesid := GenerateIndexesId()
+
+							sourceindexes.Id = indexesid
+							spannerindexes.Id = indexesid
+
+							conv.SrcSchema[sourcetable.Name].Indexes[sourcei] = sourceindexes
+							conv.SpSchema[spannertable.Name].Indexes[spanneri] = spannerindexes
+
+						}
+
+					}
+				}
+
+				updateSpannerTableIndexKeyOrder(spannertable)
+				updateSourceTableIndexKeyOrder(sourcetable)
+
+				conv.SrcSchema[sourcetablename] = sourcetable
+				conv.SpSchema[spannertablename] = spannertable
+				break
+			}
+		}
+	}
+
+}
+
+// updateSpannerTableIndexKeyOrder Update Primary Key Order as columnId.
+func updateSpannerTableIndexKeyOrder(spannertable ddl.CreateTable) {
+
+	for i := 0; i < len(spannertable.Pks); i++ {
+		for spannercolumnname, _ := range spannertable.ColDefs {
+			if spannertable.Pks[i].Col == spannercolumnname {
+
+				o := getSpannerColumnIndex(spannertable, spannercolumnname)
+				spannertable.Pks[i].Order = o
+
+			}
+		}
+	}
+}
+
+// updateSourceTableIndexKeyOrder Update Primary Key Order as columnId.
+func updateSourceTableIndexKeyOrder(sourcetable schema.Table) {
+
+	for i := 0; i < len(sourcetable.PrimaryKeys); i++ {
+		for sourcecolumnname, _ := range sourcetable.ColDefs {
+			if sourcetable.PrimaryKeys[i].Column == sourcecolumnname {
+
+				o := getSourceColumnIndex(sourcetable, sourcecolumnname)
+				sourcetable.PrimaryKeys[i].Order = o
+			}
+		}
+	}
+}
+
+// getSpannerColumnIndex return columnn index as Inserted Order.
+func getSpannerColumnIndex(spannertable ddl.CreateTable, columnName string) int {
+
+	for i := 0; i < len(spannertable.ColNames); i++ {
+		if spannertable.ColNames[i] == columnName {
+			return i + 1
+		}
+	}
+	return 0
+}
+
+// getColumnIndex return columnn index as Inserted Order.
+func getSourceColumnIndex(sourcetable schema.Table, columnName string) int {
+
+	for i := 0; i < len(sourcetable.ColNames); i++ {
+		if sourcetable.ColNames[i] == columnName {
+			return i + 1
+		}
+	}
+	return 0
+}
+
+func GenerateId() string {
+
+	sessionState := session.GetSessionState()
+
+	counter, _ := strconv.Atoi(sessionState.Counter.ObjectId)
+
+	counter = counter + 1
+
+	sessionState.Counter.ObjectId = strconv.Itoa(counter)
+	return sessionState.Counter.ObjectId
+}
+
+func GenerateTableId() string {
+	tablePrefix := "t"
+	id := GenerateId()
+	tableId := tablePrefix + id
+	return tableId
+}
+
+func GenerateColumnId() string {
+
+	columnPrefix := "c"
+	id := GenerateId()
+	columnId := columnPrefix + id
+	return columnId
+}
+
+func GenerateForeignkeyId() string {
+
+	foreignKeyPrefix := "f"
+	id := GenerateId()
+	foreignKeyId := foreignKeyPrefix + id
+	return foreignKeyId
+}
+
+func GenerateIndexesId() string {
+
+	indexesPrefix := "i"
+	id := GenerateId()
+
+	indexesId := indexesPrefix + id
+	return indexesId
+}
```

Comment:
We can define these prefixes as constants and use directly with GenerateId(). There is no need of these seperate methods